src/client/entity-editor/validators/common.ts

```diff
@@ -184,7 +184,8 @@ export function validateSubmissionSection(
 	);
 }
 
-export function validateAuthorCreditRow(row: any): boolean {
+export function validateAuthorCreditRow(row: any, isOptional = false): boolean {
+	if (!getIn(row, ['author', 'id'], null) && !get(row, 'name', null) && isOptional) { return true; }
```

Comment:
I'm not sure I understand why we check this in the AC row.
Shouldn't the check be done in validateAuthorCreditSection instead?

Reply:
This check is mainly for first row of AC, since in inline editor first row always exist but might not contain any author. which should be valid if optional checkbox is ticked.